addons/l10n_ma/data/l10n_ma_chart_data.xml

```diff
@@ -1,12 +1,12 @@
 <?xml version="1.0" encoding="ISO-8859-15"?>
 <odoo>
-        <!--     
```

Comment:
I don't get it...

and the first line still says it's latin-9, not UTF8 
`<?xml version="1.0" encoding="ISO-8859-15"?>`

Reply:
Yeah my bad, I just changed the encoding without changing the content. I don't even remember why I did this on this PR. I also have another commit on another branch that puts this in csv. Maybe I should remove this from here?

Reply:
yes indeed :-)